### DISCUSSION TITLE
Seplos BMS library

### DISCUSSION THREAD

**Discussion by M4RV1NC** [syssi/esphome-seplos-bms#119] [Q&A]
Hello,

Is it possible to have a separate seplos library that can be intregrate with others project ? I don't using HA... and ESP home

Thanks for your job

Vince

**Comment by M4RV1NC** [comment]
[SEPLOS-3.0RS485BMS Modbus RTU Protocol.pdf](https://github.com/user-attachments/files/18279908/SEPLOS-3.0RS485BMS.Modbus.RTU.Protocol.pdf)

to help, I will develop my own sketch without any response... enjoy.

**Reply by syssi** [reply]
Feel free to extract the important parts. Providing and maintaining a library is beyond my abilities.
